sim/dex.ts

```diff
@@ -472,6 +480,35 @@ export class ModdedDex {
 		return learnsetData;
 	}
 
+	getDescs(table: keyof TextTableData, id: ID, dataEntry: AnyObject) {
+		if (dataEntry.shortDesc) {
+			return {
+				desc: dataEntry.desc,
+				shortDesc: dataEntry.shortDesc,
+			};
+		}
+		const entry = this.loadTextData()[table][id];
+		if (!entry) return null;
+		const descs = {
+			desc: '',
+			shortDesc: '',
```

Comment:
Replacing these with the `dataEntry`'s current values should fix this, right, @urkerab?

Reply:
That depends on what you want the inheritance behaviour to be... given that you added an early return for mod-supplied descriptions, I wouldn't like to guess.

Reply:
As mentioned earlier, I have no clue what the inheritance behavior should be and I'm open to suggestions. But I think we should probably at least want to support overriding `desc` while inheriting `shortDesc`.